### ISSUE TITLE
Translate and publish alert on Spanish COVID page

### ISSUE THREAD

**Issue by RLHecht** [department-of-veterans-affairs/va.gov-team#57232]
## Issue Description

When we make changes to our English COVID page, we need to send the content out for Spanish and Tagalog translation. Because it can often take up to 2 weeks to get these changes back, we need to add an alert to the Spanish and Tagalog page explaining that the page is awaiting translation. 

Currently we have a note under the blue box stating that we make changes frequently so the translated pages may not be up to date. We're going to remove this note and use the alert instead. When we get translations back and all 3 pages are in sync, we should remove the alert. 

Content to translate: 

**This page is currently awaiting translation of the most recent updates. Check back soon or go to the English page for the latest updates.**

We are making important changes to the [COVID vaccines](https://www.va.gov/health-care/covid-19-vaccine/), [boosters](https://www.va.gov/health-care/covid-19-vaccine/booster-shots-and-additional-doses/) pages and will want to add this alert to both of these Spanish pages. 

## Tasks
- [x] Translate content in Spanish (Ana)
- [x] Create alert in Drupal and place on 2 Spanish COVID pages (Randi)
- [x] Review content (Aliyah)
- [x] Publish alert and pages (Aliyah/Randi)

## Acceptance Criteria
- [x] Spanish alert is live on the site

**Comment by ABeltran77** [comment]
Here is the translated content:

Esta página está actualmente esperando la traducción de las actualizaciones más recientes. Vuelva pronto o vaya a la página en inglés para obtener las últimas actualizaciones.

**Comment by RLHecht** [comment]
@ABeltran77 We need to add a header to the alert. Can you please translate this as well?

We're currently translating the latest updates to this information

**Comment by ABeltran77** [comment]
Translated header:
Actualmente estamos traduciendo las últimas actualizaciones de esta información.

**Comment by RLHecht** [comment]
@aliyahblackmore this is ready for you. Can you please review the alert? I added it to the Spanish main vaccine page and the boosters page. And I deleted the note on the main vaccines page that was under the blue box.

**Comment by aliyahblackmore** [comment]
Thanks Randi and Ana! 

@RLHecht Randi - I'm tracking some of other changes we chatted about on Slack:

1. We're adding a link to the English version of the page on the alert. Ana translated the link text: **Obtenga más información sobre las últimas actualizaciones en la página en inglés**
2. Because we're including this link on both pages, I created another alert for the booster shots page. And I added the page type to the alert description just so we can differentiate between the two!
